### PR TITLE
Delete services

### PR DESCRIPTION
**Pull request by blackenjoy27**
### Description

The app was having trouble delete services out of the service table, our code fix the problem.

**Invited Collaborator(s) to Review
- Added person to review relating to feature/bug
- If not available, ask another reviewer on team

**What work was done?**

For the backend:
We created a function used for service_entry table (database) inside its model file, and change the migrations files so that it will be tolerate for deletion without violating the foreign key constraint.

For the frontend:
I change the variable name so that the element is referring to the correct attribute inside the object that is passed from the database.
I added the a pop message that will show up whenever a service is deleted.

**Why was this work done?**

If the user can't be able to delete a service record out of the database, not only it will impact users experience as the database gets bigger, it would also be a heavy work to maintain large database.

- What is the purpose of the feature?

The purpose of this feature is let specific role of users have the feature to delete services so that other users would not be seeing service that will no longer available.

**What feature / user story is it for?**
(Link if possible) 
**Any relevant links or images / screenshots**

https://lambda-students.slack.com/archives/C02AFKQBD2L/p1628643484202800

### Type of change
Please delete options that are not relevant.
- Fix feature 

Testing
- Has this feature been tested before conflict?
- Has this feature been tested after conflict?

Change Status
- Completed, ready to review and merge

### Checklist
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- My code has been reviewed by at least one peer
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- My changes generate no new warnings
- There are no merge conflicts

### CODE REVIEW

File: src/components/forms/AddRecipientForm.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ const { Option } = Select;
 
 function AddRecipientForm({ visible, onCreate, onCancel, households }) {
   const [form] = Form.useForm();
+  console.log('Look', households);
```
Comment (chance10113) [inline src/components/forms/AddRecipientForm.js:11]: You should remove this console log before you merge

File: src/components/forms/AddRecipientForm.js [hunk 2]
```diff
@@ -22,6 +23,8 @@ function AddRecipientForm({ visible, onCreate, onCancel, households }) {
           form
             .validateFields()
             .then(values => {
+              console.log('Households: ', households);
```
Comment (chance10113) [inline src/components/forms/AddRecipientForm.js:26]: These console logs should also be removed

**Review by chance10113 — APPROVED** [review]
As long as you remove the console logs, this code seems good!
Way to go getting the delete to work!

**Review by ahmedseragcodes — APPROVED** [review]
Non-breaking changes + solves key issue + no conflicts = > Approved. Nice job.

**Pull request merged**